src/views/3-Modeling/Modeling.vue

```diff
@@ -4,292 +4,224 @@
       id="intro-container"
       class="text-content"
     >
-      <h2 v-html="text.title1" />
-      <p v-html="text.paragraph1" />
+      <h2>Modeling!</h2>
     </div>
     <!--  figure contains all the sticky elements -->
     <figure
       ref="figure"
       class="sticky"
     >
       <div id="flubber-container">
-        <div id="flubber">
-          <svg
-            id="transform-svg-test"
-            xmlns="http://www.w3.org/2000/svg"
-            width="300px"
-            height="300px"
+        <svg
+          id="flubber"
```

Comment:
For no good reason I'm actually still using the id `"transform-svg-test"` for this svg. And right now, to work w/ the css, it does have to be within a div with the id `"flubber"`

Reply:
that fixed it 😌